src/Actions/RelativeLineNumbers.ts

```diff
@@ -11,6 +11,12 @@ export class ActionRelativeLineNumbers {
                     return Promise.resolve(false);
                 }
 
+                const lineNumbersTurnedOff = activeTextEditor.options.lineNumbers === TextEditorLineNumbersStyle.Off
```

Comment:
Since the condition is pretty simple, we cloud put it into `if` directly.
How do you think?

Reply:
👍